azext_iot/iothub/params.py

```diff
@@ -249,6 +251,78 @@ def load_iothub_arguments(self, _):
             help="MIME Type of file.",
         )
 
+    with self.argument_context("iot hub state") as context:
+        context.argument(
+            "state_file",
+            options_list=["--state-file", "-f"],
+            help="The path to the file where the state information will be stored."
+        )
+        context.argument(
+            "replace",
+            options_list=["--replace", "-r"],
+            help="If this flag is set, then the command will delete the current devices, configurations, and certificates "
+                 "of the destination hub."
+        )
+        context.argument(
+            "hub_aspects",
+            options_list=["--aspects"],
+            nargs="+",
+            arg_type=get_enum_type(HubAspects),
+            help="Hub Aspects (space-separated)."
+        )
+
+    with self.argument_context("iot hub state export") as context:
+        context.argument(
+            "replace",
+            options_list=["--replace", "-r"],
+            help="If this flag is set, then the command will overwrite the contents of the output file."
+        )
+
+    with self.argument_context("iot hub state migrate") as context:
+        context.argument(
+            "hub_name",
+            options_list=["--destination-hub", "--dh"],
+            help="Name of IoT Hub to which the origin hub will be copied."
```

Comment:
nit, "Name of IoT Hub which the origin hub **state** will be copied **to**."?

Reply:
I feel like I had an argument with Mirabai about this... but yes the to feels better